### PR TITLE
Origin/qq main

### PR DESCRIPTION
**Pull request by wgranados**
Added pytest and setup configuration so tests run on travis ci.

### CODE REVIEW

**Comment by QuiteQuiet** [comment]
Can you get rid of the commit I made before the ones relevant to the pullrequest begins?

**Comment by wgranados** [comment]
reverting the [commit](https://github.com/QuiteQuiet/PokemonShowdownBot/pull/3/commits/d9529b1c0f10c8f38e1a2db1d8f1a3b8ad8c2df8) causes the tests to fail. This is because of the missing change to the regex (adding $). I can revert the commit and add that modification to the regex in a new commit.

**Comment by QuiteQuiet** [comment]
You can rebase to fix that.

**Comment by QuiteQuiet** [comment]
Resolved the commit problems myself. Thanks for doing this so I can keep being lazy.

File: app.py [hunk 1]
```diff
@@ -239,17 +239,17 @@ def parseMessage(self, msg, roomName):
...
-    # This function has a loop that runs as long as the websocket is connected
-    psb.ws.run_forever()
-    # If we get here, the socket is closed and disconnected
-    # so we have to reconnect and restart (after waiting a bit of course, say half a minute)
-    time.sleep(30)
-    print('30 seconds since last disconnect. Retrying connection...')
-    psb.openConnection()
-    restartCount += 1
-    print('Restart Count:', restartCount)
+if __name__ == '__main':
```
Comment (QuiteQuiet) [inline app.py:242]: On second thought, it's amazing this ever worked...

Comment (wgranados) [reply app.py:242]: oops

**Pull request merged**